ref-exchange/build.sh

```diff
@@ -2,4 +2,5 @@
 set -e
 
 RUSTFLAGS='-C link-arg=-s' cargo +stable build --target wasm32-unknown-unknown --release
-cp target/wasm32-unknown-unknown/release/ref_exchange.wasm ../res/
+cd ..
+cp target/wasm32-unknown-unknown/release/ref_exchange.wasm ./res/
```

Comment:
why not common res folder outside?

Reply:
It's is a common directory. When using workspace, the current directory is the workspace directory.